queries/go/textobjects.scm

```diff
@@ -97,3 +97,25 @@
   . (_) @parameter.inner
   . ","? @_end
  (#make-range! "parameter.outer" @parameter.inner @_end))
+
+;; assignments
+(short_var_declaration
+  left: (_) @assignment.lhs
+  right: (_) @assignment.rhs @assignment.inner) @assignment.outer
+(assignment_statement
+  left: (_) @assignment.lhs
+  right: (_) @assignment.rhs @assignment.inner) @assignment.outer
+(var_spec
+  name: (_) @assignment.lhs
+  value: (_) @assignment.rhs @assignment.inner)
+(var_spec
+  name: (_) @assignment.inner
+  type: (_))
+(var_spec) @assignment.outer
```

Comment:
Instead of querying it a third time, just add the capture to the other two (same for `(const_spec)`).

Reply:
Yup, makes sense